extensions/SemanticMediaWiki/includes/specials/SMW_SpecialSMWAdmin.php

```diff
@@ -84,6 +85,8 @@ public function execute( $par ) {
 		} else {
 			$refreshjob = null;
 		}
+		**/
+		$refreshjob = null; // Wikia change - we do not use job DB table for handling tasks queue (SUS-1305)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)